init

```diff
@@ -62,7 +62,7 @@ fi
 # mapped to the /opt directory of these scripts
 etcdctl get /environment/services
 if [[ $? = 4 ]]; then
-  etcdctl set /environment/services "sumologic datadog"
+  etcdctl set /environment/services "sumologic datadog splunk"
```

Comment:
this can be removed - users will just overwrite `/environment/services` with `"sumologic datadog splunk"` in their cluster configs if they want splunk